README.md

```diff
@@ -183,15 +183,16 @@ More ready-to-use examples can be found in the [Example](https://mosecorg.github
 
 ## Deployment
 
-- This may require some shared memory, remember to set the `--shm-size` flag if you are using docker.
-- This service doesn't require Gunicorn or NGINX, but you can certainly use the ingress controller. BTW, it should be the PID 1 process in the container since it controls multiple processes.
+- If you're looking for a GPU base image with `mosec` installed, you can check the official image [`mosecorg/mosec`](https://hub.docker.com/r/mosecorg/mosec). For the complex use case, check out [envd](https://github.com/tensorchord/envd).
```

Comment:
What is "the complex use case"? Did you mean more docker images?

Reply:
It means a GPU docker image with lots of dependencies (pip, apt, etc.)